README.md

```diff
@@ -1,70 +1,285 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
```

Comment:
- [ ] You have done well using the template for your README but you still need to update ALL your sections to be more descriptive. Currently you are still using the dummy data for sections. Kindly update your README accordingly.

Reply:
Hi, I think the screenshot is the readme of dev branch. I have updated the new template of readme in the setup branch because the previous reviewer asked me to change it. Could you kindly check the readme in the setup branch pls?